packages/zip-it-and-ship-it/src/runtimes/node/bundlers/esbuild/bundler.ts

```diff
@@ -60,11 +60,13 @@ const includedFilesToEsbuildExternals = async (includedFiles: string[], baseDir:
 
     if (hasMultipleGlobs) {
       const resolved = await glob(pattern, {
-        noglobstar: true,
+        globstar: false,
         cwd: baseDir,
       })
 
-      result.push(...resolved)
+      // esbuild expects relative paths, but tinyglobby uses `posix.normalize()` which strips leading `./`
```

Comment:
Credit to @serhalp for figuring this one out

Reply:
This comment is about tinyglobby (from my other PR), but here we're switching to fast-glob.

AFAICT from reading the fast-glob source and tests, this shouldn't be needed, I think. I wonder if this is resulting in `././foo` which happens to still work fine?

Reply:
Yeah let's figure out if that's the case and avoid the duplicated segments if possible.

Reply:
Without this code the test `included_files with multiple glob stars are correctly resolved before passing to esbuild` fails. I'll update the comment though.